maps/src/main/scala/dsentric/codecs/std/DValueCodecs.scala

```diff
@@ -68,6 +68,18 @@ trait DValueCodecs {
         IntegerDefinition(minimum = Some(Long.MinValue), maximum = Some(Long.MaxValue))
     }
 
+  implicit def oneCodec: DCodec[1] = new DValueCodec[1] {
```

Comment:
I would make this a val as we dont want to create one for each set dcodec

Reply:
fixed